onnxruntime/test/util/scoped_env_vars.cc

```diff
@@ -43,20 +43,24 @@ Status SetEnvironmentVar(const std::string& name, const optional<std::string>& v
 Status GetEnvironmentVar(const std::string& name, optional<std::string>& value) {
   constexpr DWORD kBufferSize = 32767;
 
-  char buffer[kBufferSize];
-
-  const auto char_count = GetEnvironmentVariableA(name.c_str(), buffer, kBufferSize);
-  if (char_count > 0) {
-    value = std::string{buffer, buffer + char_count};
-    return Status::OK();
+  std::string buffer(kBufferSize, '\0');
```

Comment:
This change is to reduce stack size requirement, to make VC++ static analyzer happy.